stable/etcd-operator/values.yaml

```diff
@@ -31,3 +31,4 @@ cluster:
       storageType: PersistentVolume
       pv:
         volumeSizeInMB: 512
+  extraArgs: {}
```

Comment:
Can you put reference to the upstream link with some examples and also add some commented out examples here?